asv/benchmark.py

```diff
@@ -574,7 +568,7 @@ def do_run(self):
 
     def do_profile(self, filename=None):
         def method_caller():
-            run(*params)
+            run(*params) # noqa see #1020
```

Comment:
When using `noqa` we need to specify the exact error code we want to ignore. Also, if besides pointing to the issue you can write a very short sentence on what's the problem  and why this should be skipped, that would be useful